plugin-server/src/worker/plugins/transpile.ts

```diff
@@ -0,0 +1,20 @@
+import { transform } from '@babel/standalone'
```

Comment:
can we move this file inside of a new `frontend_plugins` dir so things are more clear?